toolkit/scripts/pkggen.mk

```diff
@@ -171,20 +151,41 @@ $(RPMS_DIR):
 	@touch $@
 endif
 
-$(STATUS_FLAGS_DIR)/build-rpms.flag: $(workplan) $(chroot_worker) $(go-pkgworker)
-	rm -f $(LOGS_DIR)/pkggen/failures.txt && \
-	$(MAKE) --silent -f $(workplan) go-pkgworker=$(go-pkgworker) CHROOT_DIR=$(CHROOT_DIR) chroot_worker=$(chroot_worker) SRPMS_DIR=$(SRPMS_DIR) RPMS_DIR=$(RPMS_DIR) pkggen_local_repo=$(pkggen_local_repo) LOGS_DIR=$(LOGS_DIR) TOOLCHAIN_MANIFESTS_DIR=$(TOOLCHAIN_MANIFESTS_DIR) GOAL_PackagesToBuild && \
-	{ [ ! -f $(LOGS_DIR)/pkggen/failures.txt ] || \
-		$(call print_error,Failed to build: $$(cat $(LOGS_DIR)/pkggen/failures.txt)); } && \
+$(STATUS_FLAGS_DIR)/build-rpms.flag: $(cached_file) $(chroot_worker) $(go-scheduler) $(go-pkgworker) $(depend_STOP_ON_PKG_FAIL) $(CONFIG_FILE) $(depend_CONFIG_FILE)
```

Comment:
>$(CONFIG_FILE) $(depend_CONFIG_FILE) [](start = 128, length = 36)

Question: considering the `$(if $(CONFIG_FILE),--base-dir=$(CONFIG_BASE_DIR))`  part below, do we really depend on the config file here? Or should that check be done against `CONFIG_BASE_DIR`?

Reply:
The `$(depend_CONFIG_FILE)` tracks if the `CONFIG_FILE` value has changed, even if they share the same base directory